src/components/Footer.jsx

```diff
@@ -0,0 +1,70 @@
+import { useTranslation } from "react-i18next";
+import { useState, useEffect } from "react"
+import map from "../../public/assets/map.png";
+import mapPin from "../../public/assets/mapPin.svg";
+import "/node_modules/flag-icons/css/flag-icons.min.css";
+import "./Footer.css";
+
+export const Footer = () => {
+
+    const [language, setLanguage] = useState("en")
+    const { t, i18n } = useTranslation();
+
+    // function to change the language
+    const changeLanguageFunc = (lng) => {
+        i18n.changeLanguage(lng);
+    };
+
+    const englishFunc = () => changeLanguageFunc("en");
+    const swedishFunc = () => changeLanguageFunc("se");
+
+    useEffect(() => {
+      setLanguage(i18n.language);
+    }, [i18n.language])
+
+    const getFlagClass = (lng) => {
+      switch (lng) {
+        case "en": 
+          return "fi-gb";
+        case "se":
+          return "fi-se";
+        default:
+          return "";
+      }
+    };
```

Comment:
Nice to use a function with a switch to choose between the flags, perhaps some comments on this would also be beneficial? 🥰